site/_posts/2021-02-16-orbitvu-fusionauth-story.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: blog-post
+title: Orbitvu chose FusionAuth for architectural flexibility, great support, and customizability
+description: Orbitvu, a product photography company, gets more flexibility and value with FusionAuth.
+author: Dan Moore
+image: blogs/become-education-story/become-education-chooses-fusionauth-for-idaas.png
+category: blog
+tags: topic-community-story
+excerpt_separator: "<!--more-->"
+---
+
+Maciej Wisniowski is a FusionAuth community member and team leader in the web development department at Orbitvu, where he is building innovative eCommerce imaging solutions. He chatted with us over email about how he and his team are using FusionAuth to meet their auth needs. 
+
+<!--more-->
+
+*This interview has been lightly edited for clarity and length.*
+
+-------
+
+**Dan:** Tell me a bit about your work as a developer.
+
+**Maciej:** I'm a full-stack web developer working as a Team Leader in Orbitvu, a company that builds innovative imaging solutions for the world of eCommerce. 
+
+Orbitvu provides both software and hardware solutions for automated product photography which means our client gets an end-to-end solution that starts with our hardware and desktop software with features like automatic background removal and then is able to effortlessly publish packshots and 360-degree interactive product presentations on the web. The latter part is my responsibility.
+
+**Dan:** What a cool idea! Do you have a link to a sample (or real) interactive product presentation? 
+
+**Maciej:** Of course, the typical 360-degree presentation made with our solutions might be similar to this presentation of the [miter saw](https://orbitvu.co/share/uEnZJgngCNobquxATcbAyf/17144/orbittour/view). We have a number of products targeted to different industries and/or product sizes. For example, our [Alphashot Micro](https://orbitvu.com/product/alphashot-micro/) is made for photography of smaller objects, eg. jewelry or watches. For the bigger objects, also human models, we have other solutions like our newest product [Fashion Studio](https://orbitvu.com/product/fashion-studio/) that is great not just for packshots but also for the videos.
+
+> [Choosing FusionAuth] opened a whole bunch of new possibilities in terms of possible architectural solutions and technologies we were able to use. 
+
+**Dan:** How do you use FusionAuth? OAuth? User management? Social sign-on? Something else?
+
+**Maciej:** We use FusionAuth as an OAuth2/OIDC/SAML server (with integrated Google sign-on) to provide login and SSO to our web applications and supplementary systems like the site with our manuals. 
+
+We are also starting to use FusionAuth internally. Orbitvu is not just a software development company but we also design and manufacture our automated photo studios. Because of that, we have a lot of different departments that utilize a number of internal systems like ERP, project management software, or Helpdesk. All of these can also benefit from FusionAuth integration.
```

Comment:
~Because of that,~ We have a lot of different departments that utilize a number of internal systems like ERP, project management software, or Helpdesk~. All of these can also~ **which all** benefit from **our** FusionAuth integration.